### PR TITLE
Fix/survey summary

### PR DESCRIPTION
**Pull request by melokda**
# Description 

Modified the survey summary page to match the rest of the app styles

# Changes 

-Deleted background questions summary view
-Changed the style of the indicator questions summary list to match the surveys list
-Deleted some unnecessary code

# Screenshots 

image

# Tests 

This code has been tested in the following ways:
- Tasks:
  - [X] Logged in to the application
  - [X] Fill out a snapshot for a new family
  - [ ] Fill out a snapshot for an existing family
  - [ ] Fill out priorities for a snapshot
  - [ ] View a family and it's priorities
- API Level:
  - [X] API Level 19
  - [ ] API Level 22
  - [X] API Level 25
- Screen Size:
  - [ ] 7" screen size
  - [X] 8" screen size
  - [X] 10" screen size

Couldn't text on 7" because I don't have a 7" emulator with play services

### CODE REVIEW

**Comment by bhylak** [comment]
Definitely looks a lot better than before, but the colors don't quite meet our style guide. (In fact, the survey selection page *also* doesn't match our style guide. I realize you just used that as a point of reference)

Take a look at the mock for the survey selection page. 

I made this quickly to show you what i mean. We should also move away from the nonstandard next/previous buttons.

image

Also it's not using the correct indicator names (#154)

**Comment by krconv** [comment]
Can you file the cards with `indicator.getTitle()` (e.g. Entertainment And Recreation) instead of `indicator.getName()` (e.g. entertainmentAndRecreation)? I know that it was like that before, but it seems like only a one line change and would make it look better IMO

**Comment by bhylak** [comment]
@krconv i think she updated that

**Comment by bhylak** [comment]
@melokda Your changes are great, but unfortunately the code that you built upon is architected terribly. There's an adapter in a view component, for instance. And there's a ton of code that does absolutely nothing now.

For example, on resume has this whole function that creates a list of names for some unknown reason and then dumps it all to the view and gets the adapter from the view and then adds a callback to the adapter within the view.

Not sure why it was architected like that in the first place, but it's a big problem. Personally I can't approve it until that's updated, but if someone else (like @krconv) disagrees then it's fine by me.

**Comment by krconv** [comment]
I'm ok with approving as long as this works and new issues aren't introduced; I think re-architecture is out of the scope of this problem

**Comment by melokda** [comment]
@bhylak @krconv Can someone please look at my changes and review or give me comments?

image

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/SurveySummaryFragment.java [hunk 1]
```diff
@@ -24,20 +27,20 @@
 
 public class SurveySummaryFragment extends AbstractSurveyFragment implements SurveySummaryAdapter.InterfaceClickListener {
```
Comment (bhylak) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/SurveySummaryFragment.java]: can you rename this class to SurveyIndicatorsSummary

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/SurveySummaryFragment.java [hunk 2]
```diff
@@ -24,20 +27,20 @@
...
 
-    SurveySummaryAdapter indicatorAdapter;
-
     ArrayList<String> indicatorNames = new ArrayList<>();
 
-    Button mSubmitButton;
-    Button mBackButton;
+    private TextView numSkipped;
+    private RecyclerView mIndicatorList;
+    private SurveySummaryAdapter mSurveySummaryAdapter;
```
Comment (bhylak) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/SurveySummaryFragment.java]: Change this adapter to IndicatorsSummaryAdapter

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/SurveySummaryFragment.java [hunk 3]
```diff
@@ -24,20 +27,20 @@
...
+
+    public enum SurveySummaryState{COMPLETE, INCOMPLETE}
```
Comment (bhylak) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/SurveySummaryFragment.java]: Change name to IndicatorQuestionState

**Pull request merged**